src/gitlab.js

```diff
@@ -78,6 +81,71 @@ class GitLab {
     });
   }
 
+  loadTriggers(proj_name) {
+    var projId = encodeURIComponent(proj_name);
+    m.request({
+      url: `${this.api_path}/projects/${projId}/triggers`,
+      method: "GET",
+      headers: {
+        "PRIVATE-TOKEN": this.private_token
+      }
+    }).then((data) => {
+      this.triggers = data;
+      return Promise.resolve(data);
+    }).catch((e) => {
+      alert(e);
+      return Promise.reject();
+    });
+  }
+  
+  loadBranchs(proj_name) {
+    if (this.api_path.length > 0){
+      this.branchs = null;
+      var proj_id = encodeURIComponent(proj_name);
+      return this.loadBranchsBase(proj_id, 1, []);
+    } else {
+      this.branchs = [];
+      return Promise.resolve(this.branchs);
+    }
+  }
+  loadBranchsBase(proj_id, page, all_branchs) {
+    const data = {
+      page: page,
+      per_page: this.per_page,
+      order_by: "name",
```

Comment:
I think that `page` and `per_page` are working, but `order_by` and `sort` aren't woking.

ref. 

* https://gitlab.com/gitlab-org/gitlab-ce/blob/v10.2.2/lib/api/branches.rb#L29
* https://gitlab.com/gitlab-org/gitlab-ce/blob/v10.2.2/lib/api/pagination_params.rb

Reply:
gitlab always sort branches by name?

Reply:
yes (at v10.2.2)
https://gitlab.com/gitlab-org/gitlab-ce/blob/v10.2.2/lib/api/branches.rb#L33

There is a possibility that it will change in the future